### PR TITLE
NAACL 2022 ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Review by mjpost — COMMENTED** [review]
NAACL volume order should be SRW, Tutorial Abstracts, then Industry.

**Comment by xinru1414** [comment]
> NAACL volume order should be SRW, Tutorial Abstracts, then Industry.

Yep will adjust the order once long and short are in

**Comment by mjpost** [comment]
How close are we to being finalized here?

File: bin/ingest_aclpub2.py [hunk 1]
```diff
@@ -134,9 +134,10 @@ def parse_conf_yaml(ingestion_dir: str) -> Dict[str, Any]:
     for key in must_have_keys:
         assert key in meta.keys(), f'{key} is missing in the conference_details.yml file'
 
-    if re.match(rf'^[a-z0-9]+$', meta['volume']) is None:
+    meta['volume_name'] = str(meta['volume_name'])
```
Comment (mjpost) [inline bin/ingest_aclpub2.py:137]: This is a strange line; what is the type before this call?

File: data/xml/2022.sem.xml [hunk 2]
```diff
@@ -0,0 +1,330 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.sem">
```
Comment (mjpost) [inline data/xml/2022.sem.xml:2]: This is the wrong venue code: it needs to be [starsem](https://aclanthology.org/venues/starsem/).

File: data/xml/2022.sigmorphon.xml [hunk 3]
```diff
@@ -0,0 +1,301 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.sigmorphon">
+  <volume id="1" ingest-date="2022-07-06">
+    <meta>
+      <booktitle>19th SIGMORPHON Workshop on Computational Research in Phonetics, Phonology, and Morphology</booktitle>
```
Comment (mjpost) [inline data/xml/2022.sigmorphon.xml:5]: The title should be “Proceedings of the 19th…” (see [here](https://preview.aclanthology.org/naacl-22-ingestion/venues/sigmorphon/)).

**Comment by mjpost** [comment]
In addition to a few minor issues, the NAACL industry track papers volume is named inconsistently with the others. [See here](https://preview.aclanthology.org/naacl-22-ingestion/venues/naacl/). It should be “Proceedings of the 2022 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies: Industry Track Papers”.

File: data/xml/2022.sem.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/2022.sem.xml:2]: I changed this.

**Review by mjpost — APPROVED** [review]

**Pull request merged**